README.md

```diff
@@ -1,2 +1,38 @@
 # hello_microverse_project
-Setting up a "Hello Microverse" project with the correct and professional structure.
+
+hello_microverse_project is a basic Microverse project to familiarize with GitHub workflow and create professional repositories.
+
+## Built With
+
+### Tech Stack
+
+__Client__
+- HTML
+- CSS
+
+### Key Features
+
+- Create a basic project structure
+- Develop a professional repository
+- Add useful commits
+
+## Getting Started
+
+### Prerequisites
+
+- Install [Git CLI](https://git-scm.com/downloads) to clone the repository.
+- Install a [code editor](https://code.visualstudio.com/download) to open the cloned repository.
+
+### Usage
+
+Once you've cloned the repository and you have it opened in your code editor, simply open the index file either through local host, or if you're using VScode simply start live server.
+
+## Authors
+
+### Jorge Abrahan Siguenza
+- GitHub: [@jorgeabrahan](https://www.github.com/jorgeabrahan)
+- Linkedin: [@Jorge Abrahan](https://www.linkedin.com/in/jorge-abrahan-siguenza-amaya-a975741b8)
+
+## License
+
+This project is __MIT__ licensed.
```

Comment:
Kindly, update your readme based on the given template as your previous code reviewer said, check [this template](https://github.com/microverseinc/readme-template)

Reply:
@binhussen what is missing from the template? I've tried to stick to the template, the thing is that because this is a really basic project there are a lot of things from the template that are not needed, for example, the live demo and others, do you want me to add those sections any way or why are you telling me to update readme based on the template?. @binhussen

Reply:
- kindly @jorgeabrahan you removed the table of content of the readme template it has to be available.